wsi_superpixel_guided_labeling/web_client/views/vue/components/ActiveLearningReview/ActiveLearningReviewContainer.vue

```diff
@@ -290,150 +290,153 @@ export default Vue.extend({
         /**********************************************************************
          * Filter superpixels based on the selected filter options
          *********************************************************************/
-        filterBySlideName(data) {
+        filterBySlideName(data, filterBy) {
             // Filter by selected slide(s)
-            const slideNames = _.pluck(this.imageItemsById, 'name');
-            if (slideNames.some((name) => store.filterBy.includes(name))) {
-                return data.filter((superpixel) => {
-                    const name = this.imageItemsById[superpixel.imageId].name;
-                    return store.filterBy.includes(name);
-                });
-            }
-            return [];
+            return data.filter((superpixel) => {
+                const name = this.imageItemsById[superpixel.imageId].name;
+                return filterBy.indexOf(name) > -1;
```

Comment:
Can we use `includes` instead of `indexOf > -1`?

Reply:
To update after our in-person conversation: Right now `indexOf` actually gives a small performance boost over `includes`, which is why we will keep this in for now. This may change once we modernize the stack with Girder 5, so we should revisit this and possibly move back to `includes` once we do.